hoomd/jit/GPUEvalFactory.h

```diff
@@ -11,6 +11,18 @@
 #include <cuda.h>
 #include <cuda_runtime.h>
 #include <nvrtc.h>
+
+#define JITIFY_PRINT_LOG 0
+#define JITIFY_PRINT_INSTANTIATION 0
+#define JITIFY_PRINT_SOURCE 0
+#define JITIFY_PRINT_LOG 0
+#define JITIFY_PRINT_PTX 0
+//#define JITIFY_PRINT_LINKER_LOG
+#define JITIFY_PRINT_LAUNCH 0
+#define JITIFY_PRINT_HEADER_PATHS 0
```

Comment:
I do not know if there would an advantage to that. NDEBUG is really used for drilling down on segfaults, because it enables `assert`s and debug information, I wouldn't want use it for anything else, as that would not be so obvious. You can, right now, log the JIT program source and error messages with `notice_level=5`.

Reply:
As long as there's a well-defined way to capture debugging information about failures here I think that's fine. It's not clear to me what information we're declining by setting all of these flags, so I'm just wondering if there should be a way to flip them for debugging.